homeworks/homework-1/app/src/components/Header.js

```diff
@@ -0,0 +1,16 @@
+import React, {Component} from "react";
+import {Image, PageHeader} from "react-bootstrap";
+import logo from "../logo.jpg";
+
+class Header extends Component {
+
+  render() {
+    return (
+      <PageHeader className="App-header">
+        <Image src={logo} className="App-logo"/>{' '}{this.props.greeting}
```

Comment:
Also if you need some space after image please specify it in styles for that image instead using spaces {' '}